osu.Game/Overlays/MusicController.cs

```diff
@@ -293,7 +294,7 @@ private void play(BeatmapInfo info, bool isNext)
                 trackManager.SetExclusive(current.Track);
                 current.Track.Start();
                 beatmapSource.Value = current;
-            });
+            }).ContinueWith(task => Schedule(() => task.ThrowIfFaulted()));

```

Comment:
this is probably not a good idea. you're scheduling a delegate even if there's no problem. you may want to look into `TaskContinuationOptions.OnlyOnFaiulted`